unsorted/interface/frontend.gui

```diff
@@ -2393,7 +2393,7 @@ guiTypes = {
 
 				guiButtonType = {
 					name = "nation_designer_button"
-					position = { x = -241 y = -107 }
+					position = { x = -5241 y = -107 }
```

Comment:
why the drastic change here?

Reply:
the button causes a crash, and I couldn't figure out why, so I just put it outside the screen.

Reply:
Lol

Reply:
So we(other devs) need to know by heart what operators we have?

Reply:
Unless we look it up directly on arc

Reply:
wrong conversation, but yes, just like eu4 you need to memorize what operators we have (which is like 2x of vanilla) or you can just save the list somewhere.